osu.Framework.Tests/Visual/Platform/TestSceneBorderless.cs

```
@@ -85,13 +85,13 @@ private void load(FrameworkConfigManager config, GameHost host)
                 AddStep("switch to borderless", () => windowMode.Value = WindowMode.Borderless);
                 AddAssert("check window position", () => new Point(window.Position.X, window.Position.Y) == display.Bounds.Location);
                 AddAssert("check window size", () => new Size(window.Size.Width, window.Size.Height) == display.Bounds.Size, desc2);
-                AddAssert("check current screen", () => window.CurrentDisplayBindable.Value.Index == display.Index);
+                AddAssert("check current screen", () => window.CurrentDisplayBindable.Value!.Index == display.Index);
```

Let this fail gracefully, use null forgiving operator. Ditto for all `!` in tests.